From 9f8ba869af672d91dff264dd46f43ff1cc91a3aa Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Sat, 14 Aug 2010 11:36:14 +0000
Subject: [PATCH] t/Makefile: Create test-results dir for smoke target
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change the smoke target to create a test-results directory. This was
done implicitly by the test-lib before my "test-lib: Don't write
test-results when HARNESS_ACTIVE" patch, but after that smoking from
the pu branch hasn't worked.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 t/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/t/Makefile b/t/Makefile
index b81fb91..d9c67f5 100644
--- a/t/Makefile
+++ b/t/Makefile
@@ -58,7 +58,7 @@ uname_M := $(shell sh -c 'uname -m 2>/dev/null || echo unknown')
 test-results:
 	mkdir -p test-results
 
-test-results/git-smoke.tar.gz:
+test-results/git-smoke.tar.gz: test-results
 	$(PERL_PATH) ./harness \
 		--archive="test-results/git-smoke.tar.gz" \
 		$(T)
-- 
1.7.2.1.338.ge1a5e

